<?xml version="1.0"?>
<launch>
    <node pkg="tl_detector" type="tl_image_extractor.py" name="tl_image_extractor" output="screen" cwd="node"/>
  
    <param name="export_directory" type="str" value="/home/student/CarND-Capstone/export"/>
    <param name="export_filename" type="str" value="tfl_"/>
    <param name="export_rate" type="int" value="1"/>
    <param name="export_encoding" type="str" value="bgr8"/>
    <param name="export_show_image" type="bool" value="True"/>
</launch>
